WinUIGallery/Samples/Data/IconsData.json

```diff
@@ -1,4 +1,23 @@
 [
+  {
+    "Code": "E18A",
```

Comment:
Shouldnt they have the "symbol-icon" tag?

Reply:
Good catch! Thanks!